### PR TITLE
DOC-10470: Modified ramQuota flag for bucket creation

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 1]
```diff
@@ -158,10 +158,10 @@ To _edit_ the bucket, the same endpoint is used, but with the bucket name specif
 ----
 curl -v -X POST http://10.143.201.101:8091/pools/default/buckets/testBucket \
 -u Administrator:password \
--d ramQuotaMB=512
+-d ramQuota=512
 ----
 
-The value of the `ramQuotaMB` parameter (described below), is hereby increased to `512` megebytes.
+The value of the `ramQuota` parameter (described below), is hereby increased to `512` megebytes.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: This should be MiB not megebytes.
Similar for line 141, MiB not megabytes (can't comment on lines outside changes).

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 2]
```diff
@@ -186,15 +186,15 @@ The following example creates a bucket, named `testBucket`, whose type is _ephem
...
 
 If successful, the call returns a `202 Accepted` notification.
 No object is returned.
 
-[#ramquotamb]
-=== ramQuotaMB
+[#ramQuota]
+=== ramQuota
 
 The amount of memory to be allocated to the bucket, per node, in megabytes.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: MiB not megabytes

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 3]
```diff
@@ -203,9 +203,9 @@ For information on per node memory configuration, see the page for xref:manage:m
 
 A value for `ramQuota` _must_ be specified: the value _can_ be modified, following bucket-creation.
 
-An incorrect memory-specification returns a notification such as `{"ramQuotaMB":"RAM quota cannot be less than 100 MB"}`.
+An incorrect memory-specification returns a notification such as `{"ramQuota":"RAM quota cannot be less than 100 MB"}`.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: nitpick, but 100 MiB not 100MB

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 4]
```diff
@@ -203,9 +203,9 @@ For information on per node memory configuration, see the page for xref:manage:m
...
 
-[#example-ramquotamb-create]
+[#example-ramQuota-create]
 ==== Example: Specifying a Memory Quota, when Creating
 
 The following example creates a Couchbase bucket, named `testBucket` and assigns it `256` megabytes of memory.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: MiB not megabytes

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 5]
```diff
@@ -186,15 +186,15 @@ The following example creates a bucket, named `testBucket`, whose type is _ephem
...
 If successful, the call returns a `202 Accepted` notification.
 No object is returned.
 
-[#ramquotamb]
-=== ramQuotaMB
+[#ramQuota]
+=== ramQuota
 
 The amount of memory to be allocated to the bucket, per node, in megabytes.
 The minimum amount is 100 MB.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: 100 MiB not 100MB

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 6]
```diff
@@ -214,23 +214,23 @@ The following example creates a Couchbase bucket, named `testBucket` and assigns
...
 Note that the bucket is of type `couchbase` by default.
 
 If successful, the call returns a `202 Accepted` notification.
 No object is returned.
 
-[#example-ramquotamb-edit]
+[#example-ramQuota-edit]
 ==== Example: Specifying a New Memory Quota, when Editing
 
 The following example assigns a new memory quota, of `512` megabytes, to the existing bucket `testBucket`.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: MiB not megabytes

**Review by Peter-Searby — CHANGES_REQUESTED** [review]
Apologies for the nitpicking. Should be perfect with megabytes -> MiB and MB -> MiB in the cases I've suggested.

**Comment by RichardSmedley** [comment]
> Apologies for the nitpicking. Should be perfect with megabytes -> MiB and MB -> MiB in the cases I've suggested.

We're docs, we _love_ nitpicking for correctness.   :-)

File: modules/rest-api/pages/rest-bucket-create.adoc (hunk 6 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-bucket-create.adoc]: Thanks, I've changed all refs to MiB.

File: modules/rest-api/pages/rest-bucket-create.adoc [hunk 7]
```diff
@@ -196,19 +196,19 @@ No object is returned.
...
-The minimum amount is 100 MB.
+The amount of memory to be allocated to the bucket, per node, in MiB.
+The minimum amount is 100 MiB.
 The maximum amount is the total Data Service memory quota configured per node, minus the amount already assigned to other buckets.
 For information on per node memory configuration, see the page for xref:manage:manage-settings/general-settings.adoc[General] Settings.
 
 A value for `ramQuota` _must_ be specified: the value _can_ be modified, following bucket-creation.
 
-An incorrect memory-specification returns a notification such as `{"ramQuota":"RAM quota cannot be less than 100 MB"}`.
+An incorrect memory-specification returns a notification such as `{ramQuota":"RAM quota cannot be less than 100 MiB"}`.
```
Comment (Peter-Searby) [inline modules/rest-api/pages/rest-bucket-create.adoc]: The " symbol shouldn't have been removed here

**Review by Peter-Searby — CHANGES_REQUESTED** [review]
"megebytes" -> MiB wasn't fixed on line 164 and I don't see why the " symbol was removed from line 206

File: modules/rest-api/pages/rest-bucket-create.adoc (hunk 7 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-bucket-create.adoc]: Thank you. I have fixed.

**Review by Peter-Searby — APPROVED** [review]

**Pull request merged**